package.json

```
@@ -20,8 +20,8 @@
     "dist": "tsx scripts/dist.ts",
     "feature-init": "tsx scripts/feature-init.ts",
     "format": "npx prettier --write .",
-    "schema-defs:write": "npm run schema-defs -- --out ./schemas/defs.schema.json",
-    "schema-defs": "ts-json-schema-generator --tsconfig ./tsconfig.json --type WebFeaturesData --path ./types.ts --id defs",
+    "schema:write": "npm run schema -- --out ./schemas/data.schema.json",
+    "schema": "ts-json-schema-generator --tsconfig ./tsconfig.json --type WebFeaturesData --path ./types.ts",
```

This is no longer needed either (it has its own name one level down in definitions).

```suggestion
    "schema": "ts-json-schema-generator --tsconfig ./tsconfig.json --path ./types.ts",
```